conda_build/conda_interface.py

```diff
@@ -135,3 +136,160 @@ def which_prefix(path):
             # we cannot chop off any more directories, so we didn't find it
             return None
         prefix = dirname(prefix)
+
+if parse_version(conda.__version__) >= parse_version("4.3"):
+    from conda.exports import FileMode, NodeType
```

Comment:
There is already some code in the .travis.yml that will install conda canary. Since 4.3 is already in canary can I use that to test this?